supervisor/supervisor.go

```diff
@@ -141,23 +141,22 @@ func (sv *Supervisor) getHyperPod(container string, spec *specs.Spec) (hp *Hyper
 		return nil, fmt.Errorf("The container %s is already existing", container)
 	}
 	for _, ns := range spec.Linux.Namespaces {
-		if ns.Path != "" {
-			if strings.Contains(ns.Path, "/") {
-				return nil, fmt.Errorf("Runv doesn't support path to namespace file, it supports containers name as shared namespaces only")
+		if ns.Type == "network" && len(ns.Path) > 0 {
+			pidexp := regexp.MustCompile(`/proc/(\d+)/ns/net`)
```

Comment:
Please check if this path is always ns files under the /proc, if not, please compare the inode of two files.